toolkit/tools/pkg/imagecustomizerlib/cleanUpImage.go

```diff
@@ -0,0 +1,37 @@
+package imagecustomizerlib
+
+import (
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/logger"
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/safechroot"
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/shell"
+	"github.com/sirupsen/logrus"
+)
+
+func cleanTdnfCache(imageChroot *safechroot.Chroot) error {
```

Comment:
I'd probably move this function to `customizepackages.go` so it sits alongside the other tdnf helper functions.

Reply:
Moved to `customizepackages.go`